task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +101,8 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+	//  throw new Error('Not implemented');
+	return ((a < b + c) && (b < a + c) && (c < b + a)) ? true : false;
```

Comment:
we can return exactly ((a < b + c) && (b < a + c) && (c < b + a)) with the same result
Please, see https://javascript.info/logical-operators

Reply:
ok thanks